WooCommerce/Classes/ViewRelated/Products/ProductsViewController.swift

```diff
@@ -434,12 +434,25 @@ private extension ProductsViewController {
         }
     }
 
-    @objc func sortButtonTapped() {
-        UIAlertController.presentSortProductsActionSheet(viewController: self, onSelect: { [weak self] sortOrder in
-            self?.sortOrder = sortOrder
-        }, onCancel: { [weak self] in
-            self?.dismiss(animated: true, completion: nil)
-        })
+    @objc func sortButtonTapped(button: UIButton) {
```

Comment:
Small nit: I would tend to use the name `sender`, like in Apple's methods.
```suggestion
    @objc func sortButtonTapped(sender: UIButton) {
```

Reply:
yup! updated in: https://github.com/woocommerce/woocommerce-ios/pull/2156/commits/4de83cede6d2af6aca03a54dbc69a5620b3b39df